src/pages/ReimbursementAccount/ReimbursementAccountForm.js

```diff
@@ -33,6 +34,11 @@ class ReimbursementAccountForm extends React.Component {
         BankAccounts.resetReimbursementAccount();
     }
 
+    getErrorMessage() {
+        const latestErrorMessage = ErrorUtils.getLatestErrorMessage(this.props.reimbursementAccount);
+        return this.props.reimbursementAccount.error || (typeof latestErrorMessage === 'string' ? latestErrorMessage : '');
```

Comment:
I'm checking for type=string, since we set `errors[key] = true` [here](https://github.com/Expensify/App/blob/3c097924a3403d1db36f3e8fded9b668f36143b3/src/pages/ReimbursementAccount/ValidationStep.js#L109) and there were propType errors.